projects/testing/src/lib/testing.module.spec.ts

```diff
@@ -112,6 +113,43 @@ describe('TranslateTestingModule', () => {
         });
       });
     });
+
+    describe('withPlaceholderTranslations()', () => {
```

Comment:
This tests mimics the style of other tests in this file. I think it could be simplified a bit, tbh.